app/scripts/lib/ppom/ppom-util.ts

```diff
@@ -63,13 +62,11 @@ export async function validateRequestWithPPOM({
 
     const normalizedRequest = normalizePPOMRequest(request);
 
-    const ppomResponse = isSecurityAlertsAPIEnabled()
-      ? await validateWithAPI(ppomController, chainId, normalizedRequest)
-      : await validateWithController(
-          ppomController,
-          normalizedRequest,
-          chainId,
-        );
+    const ppomResponse = await validateWithAPI(
+      ppomController,
+      chainId,
+      normalizedRequest,
+    );
```

Comment:
Hey @vinistevam : is there also plan to remove code no longer used like `validateWithController` and PPOMController also can be removed from extension now.